src/doc/python_scripting/visit_utils.rst

```diff
@@ -0,0 +1,134 @@
+visit_utils
+=============
+
+visit_utils is a pure python module distributed along with Visit_'s
```

Comment:
Just a quick FYI before I continue review...we no longer [split sentences across lines](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/ContributingToDocumentation/index.html#about-line-length).